### PR TITLE
[bitnami/matomo] add independent configuration of cronjob resources

### LINKED ISSUE #23690: Ability to setup custom resources for cronjobs
### Name and Version

bitnami/matomo 

### What is the problem this feature will solve?

Currently the resources assigned to cronjobs are the same as the one of the main pod, without the possibility to do override it: https://github.com/bitnami/charts/blob/main/bitnami/matomo/templates/cronjob.yaml#L113

There is a problem running these cronjobs when you are using RWO volumes, like mentioned on [this issue](https://github.com/bitnami/charts/issues/20568). A workaround is to setup a `podAffinity` but it is not bullet proof as it will not work if the node on which the main matomo pod is running is already full. Removing the resources assigned to the cronjobs would then ensure they got executed. Not an ideal solution to have cronjobs without assigned resources, but at least it would work

### What is the feature you are proposing to solve the problem?

Default behavior would still be to inherit default resources assigned to matomo, but with the possibility to override it

### What alternatives have you considered?

_No response_

### PR DESCRIPTION
**Pull request by reiSh6phoo9o**
### Description of the change

Adds the ability to independently configure resources for cronjobs.

### Benefits

This fixes a problem when using RWO storage, where you need to ensure cronjobs can get scheduled on the same node as the main matomo deployment.

### Possible drawbacks

None

### Applicable issues

- fixes #23690

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/matomo/Chart.yaml [hunk 1]
```diff
@@ -39,4 +39,4 @@ maintainers:
 name: matomo
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/matomo
-version: 5.3.2
+version: 5.4.2
```
Comment (carrodher) [inline bitnami/matomo/Chart.yaml:42]:
```suggestion
version: 5.4.0
```

File: bitnami/matomo/Chart.yaml [hunk 2]
```diff
@@ -39,4 +39,4 @@ maintainers:
 name: matomo
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/matomo
-version: 5.4.1
+version: 5.4.2
```
Comment (joancafom) [inline bitnami/matomo/Chart.yaml:42]: There probably was a collision with versioning when integrating more recent changes from `main` branch to this one and the version is now outdated. I believe this should be a minor bump, as you are adding a new feature

```suggestion
version: 5.5.0
```

**Review by joancafom — COMMENTED** [review]
Thanks for your changes! I left you a small comment as a review, please take a look at it whenever you have the chance 😁

**Review by joancafom — APPROVED** [review]
LGTM! Thanks for your changes 🚀

**Comment by viceice** [comment]
This is a Breaking Change! Now the normal resources are no longer applied when no resource for the jobs are defined 😞 

This should be reverted or a fallback to the global resources should be added as fix.

**Pull request merged**